### DISCUSSION TITLE
Any ideas for buying/making an engram keycap set?

### DISCUSSION THREAD

**Discussion by qubitz** [binarybottle/engram-layout#43] [Q&A] [answered]
Seems like there isn't any engram keycap sets on the market. Does anyone here have engram keycaps with the proper symbol combos? I'm looking to buy/make a new keycap set and open to ideas.

**Comment by Tristen-Sinanju** [comment, marked as answer]
if you can find a regular kit or an ortho kit that uses a symmetrical cap like SDA or XDA, you can just rearrange them... but if i'm honest, it's better not to look at the keyboard when one types, so i just have blank keys on all my keyboards

**Comment by mcmam** [comment]
https://yuzukeycaps.com/ now supports a custom keycap set at a good price. I made an engram set there, and would recommend it.
